src/Compilers/CSharp/Test/Semantic/Semantics/RawInterpolationTests.cs

```diff
@@ -314,6 +314,58 @@ static void Main(string[] args)
                             world }.
                             """""" );
     }
+}";
+        CreateCompilation(source).VerifyDiagnostics(
+            // (12,1): error CS8999: Line does not start with the same whitespace as the closing line of the raw string literal.
+            // {
+            Diagnostic(ErrorCode.ERR_LineDoesNotStartWithSameWhitespace, "{").WithLocation(12, 1));
+    }
+
+    [Fact]
+    public void TwoInserts02_A()
+    {
+        string source =
+@"using System;
```

Comment:
I'm surprised you're not using a raw string for this test too :).

Reply:
:D.  when in rome.  I would be ok making a larger change to update these later.  Esp. if we may consider taking this backwards so we get the breaking change as soon as possible.